README.md

```diff
@@ -6,16 +6,31 @@
 Okta Spring Boot Starter
 ========================
 
-Okta's Spring Boot Starter will enable your Spring Boot application to work with Okta via OAuth 2.0/OIDC.  Jump to our [quickstart](https://developer.okta.com/quickstart/#/angular/java/spring) to see how to configure various clients or follow along below to use curl.
+Okta's Spring Boot Starter will enable your Spring Boot application to work with Okta via OAuth 2.0/OIDC.
 
-**NOTE:** This library works with Spring Boot 2.2+. If you need support for Spring Boot 1.5.x, use version version 0.6.
+**NOTE:** This library works with Spring Boot 2.2+. If you need support for Spring Boot 1.5.x, use version 0.6.
 
 ## What you need
 
 * An Okta account (sign up for a [forever-free developer account](https://developer.okta.com/signup/))
 * An OIDC application (typically a 'SPA' application)
 * An [access token](https://developer.okta.com/docs/api/resources/oauth2.html)
 
+## Quickstart
+
+* Create a Springboot Application with [Spring initializr](https://start.spring.io/):
+```bash
+curl https://start.spring.io/starter.zip -d dependencies=web,okta -d javaVersion=11 -o <<yourProjectName>>.zip
+```
+* Configure it with [Okta CLI](https://github.com/oktadeveloper/okta-cli/blob/master/README.md):
+```bash
+unzip <<yourProjectName>>.zip && cd <<yourProjectName>> && okta apps create
```

Comment:
```suggestion

curl https://start.spring.io/starter.tgz \
  -d dependencies=web,okta \
  -d baseDir=<yourProjectName> | tar -xzvf -
  
cd <yourProjectName>
```

* Configure it with [Okta CLI](https://github.com/oktadeveloper/okta-cli/blob/master/README.md):
```bash
okta apps create
```

Reply:
Java 11 is now the default on start.spring.io

Reply:
looks like zip is the only archive format supported (can't find tgz in spring doc at https://docs.spring.io/initializr/docs/0.6.0.RELEASE/reference/html/command-line.html).

Reply:
addressed.